ouroboros-network-testing/src/Ouroboros/Network/Testing/Utils.hs

```diff
@@ -113,6 +120,64 @@ renderRanges r n = show lower ++ " -- " ++ show upper
     lower = n - n `mod` r
     upper = lower + (r-1)
 
+--
+-- Tracing tools
+--
+
+data WithName name event = WithName {
+    wnName  :: name,
+    wnEvent :: event
+  }
+  deriving (Show, Functor)
+
+data WithTime event = WithTime {
+  wtTime  :: Time,
+  wtEvent :: event
+  }
+  deriving (Show, Functor)
```

Comment:
Could you use the same indentation style and in `WithName`

Reply:
done